src/main/java/org/osc/manager/rest/server/api/DomainApis.java

```diff
@@ -40,19 +50,55 @@
 public class DomainApis {
 
     private static final Logger logger = Logger.getLogger(DomainApis.class);
+    private EntityManager em;
+    private TransactionControl txControl;
+
+    /**
+     * init the DB
+     *
+     *
+     */
+    public void init(EntityManager em, TransactionControl txControl) throws Exception {
+        this.em = em;
+        this.txControl = txControl;
+    }
 
     /**
      * Creates the Policy for a given domain
      *
      * @return policy
      */
-    @Path("/{domainid}")
+    @Path("/{domainid}/policies")
     @POST
     public PolicyEntity createPolicy(@PathParam("domainid") Long domainid, PolicyEntity entity) {
```

Comment:
All create/POST apis should just return the id of the created entity rather than the entire object.

Reply:
ok